src/template/SubpageTabs.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function() {
+  var menu, menuToggle, toggleButton;
+
+  menu = document.getElementById("dg_tabs");
```

Comment:
const menu = 
const menuToggle = 
const toggleButton =

Reply:
Removed multi-var in favor of this style for variables. Update visible in most recent commit.
@martypowell